api-specificatie/common.yaml

```diff
@@ -556,6 +570,10 @@ components:
           minimum: 1
           maximum: 12
           example: 5
+    Datum_onvolledig:
+      deprecated: true
+      allOf:
```

Comment:
Werkt allOf met alleen een $ref bij code generatie goed? Ik heb eerder begrepen dat een allOf altijd begrepen wordt door code generatoren als class extensie. Dus dat er naast de $ref ook een object moet zijn met ten minste één property.

Reply:
Ik had het getest met nswagstudio en daar gaat het goed. Voor de zekerheid ook even getest met openapi-generator-cli die we gebruiken voor de SDK's. Daar gaat het helaas niet goed. Ik zal deze aanpassing terugdraaien

Reply:
nu hebben we dus geen DatumOnvolledig meer. Dit was wel gewenst toch?

Reply:
Ik was die dag niet meer wakker toen ik m'n test deed. Ik ging code genereren met common.yaml dat in principe nooit gaat gebeuren. Ik heb nu een goede test gedaan en ik zie dat het wel werkt, dus weer terug gezet